meson_options.txt

```diff
@@ -501,6 +501,8 @@ option('llvm-fuzz', type : 'boolean', value : false,
        description : 'build against LLVM libFuzzer')
 option('kernel-install', type: 'boolean', value: true,
        description : 'install kernel-install and associated files')
+option('ukify', type: 'boolean', value: true,
```

Comment:
any reason this needs an option? it's just a script without any dependency or side effects, no?

Reply:
The same is true for `kernel-install`. I expect that some folks will be unhappy that it's in python, so I added the option to match surrounding code.